tests/e2e/app/basic-login/autoscript.js

```diff
@@ -15,6 +15,23 @@
 
   console.log('Configure the SDK');
   forgerock.Config.set({
+    middleware: [
+      (req, action, next) => {
+        switch (action.type) {
+          case 'START_AUTHENTICATE':
+            if (action.payload.type === 'service' && typeof action.payload.tree === 'string') {
+              console.log('Starting authentication with service');
+            }
+            break;
+          case 'AUTHENTICATE':
+            if (action.payload.type === 'service' && typeof action.payload.tree === 'string') {
```

Comment:
the same as this, maybe the if should wrap the switch?

Reply:
👍  good point. I get a little lazy in the test files. I'll fix it.